.github/workflows/welcome-new-contributors.yaml

```diff
@@ -0,0 +1,25 @@
+name: 'Welcome New Contributors'
+
+on:
+  issues:
+    types: [opened]
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  welcome-new-contributor:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Greet the contributor'
+        uses: garg3133/welcome-new-contributors@v1.2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: >
+            Hey @contributor_name! Thanks for opening your first issue. We appreciate your contribution and welcome you to our community! 
+            We are glad to have you here and to have your input on Contour. 
+            You can also join us on [our mailing list](https://groups.google.com/g/project-contour) and [in our channel](https://kubernetes.slack.com/archives/C8XRH2R4J) 
+            in the [Kubernetes Slack Workspace](https://communityinviter.com/apps/kubernetes/community)
+          pr-message: >
+            Hi @contributor_name! And thank you for opening your first Pull Request. Someone will review it soon. Thank you for committing to making Contour better.
```

Comment:
Should we have "welcome to our community" sentence in PRs message too?

Reply:
Thanks for the callout. Added.